src/collections/ShowItemImageMethods.ts

```diff
@@ -0,0 +1,35 @@
+import { CollectionConfig } from 'payload/types';
+
+const ShowItemImageMethods: CollectionConfig = {
+  slug: 'showItemImageMethods',
+  fields: [
+    {
+        name: "Bildmethode",
+        type: "select",
+        label: "Show Item Image Methods",
+        hasMany: true,
+        admin: {
+            isClearable: true,
+            isSortable: true,
+          description:
+            "Bitte wählen Sie die Methode aus, die Sie für das Anzeigen von Bildern verwenden möchten.",
+        },
+        options: [
+            {
+              label: 'Metallic Paint',
+              value: 'metallic_paint',
+            },
+            {
+              label: 'Alloy Wheels',
+              value: 'alloy_wheels',
+            },
+            {
+              label: 'Carbon Fiber Dashboard',
+              value: 'carbon_fiber_dashboard',
+            },
+          ],
+      },
```

Comment:
Dieser Abschnitt muss in die Restaurant Collection. Du brauchst für diesen Fall keine Eigene Collection.

Zwei Möglichkeiten für Dropdown/Selects:
#1 Begrenzte Auswahl an Optionen (alle vom Entwickler fest einprogrammiert)
 -> Ein einfaches "select" field in die Collection mit den fest einprogrmmierten options.
#2 Optionen, die vom Admin hinzugefügt/bearbeitet werden können. Liste von Optionen kann ständig wachsen.
 -> Braucht eine zweite Colection (zum Abspechern der Optionen), diese wird dann als "relationship" in die erste Collection eingefügt.